ui/web/components/App/App.tsx

```diff
@@ -18,30 +28,46 @@ const getQuery = (): Translations => {
     }
     return enTranslations;
 }
-declare global{
-    interface Window {
-        portEvents : any;
-    }
-}  
 
 
 export const App: React.FC = () => {
 
+    // Hold the available Interval for Scheduling Information
+    const [scheduleInfo, setScheduleInfo] = React.useState<Interval>(null);
+    
     let isMobile = false;
 
     React.useEffect(() => {
         isMobile = window.matchMedia("only screen and (max-width: 500px)").matches;
-    });
+    }, []);
+
+    React.useEffect(() => {
+        let requestURL = 'https://europe.motis-project.de/?elm=requestScheduleInfo';
```

Comment:
ist das der request um zu ermitteln welche such daten legal sind?

Reply:
ja genau. Der liefert uns ein Interval zurück, in dem wir Daten vorliegen haben